drivers/storage/portworx/component/security.go

```diff
@@ -138,10 +185,9 @@ func (c *security) createPrivateKeysSecret(
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      pxutil.SecurityPXAuthKeysSecretName,
 			Namespace: cluster.Namespace,
-		},
-		StringData: map[string]string{
-			SecuritySharedSecretKey: sharedSecretKey,
-			SecuritySystemSecretKey: internalSystemSecretKey,
+		}, Data: map[string][]byte{
```

Comment:
This makes testing easier. It seems our fake driver doesn't understand StringData vs. Data, so sticking to one field helps this confusion.